backend/apps/volontulo/views/api.py

```diff
@@ -141,6 +142,17 @@ class OfferViewSet(viewsets.ModelViewSet):
 
     serializer_class = serializers.OfferSerializer
     permission_classes = (permissions.OfferPermission,)
+    filter_backends = (DjangoFilterBackend,)
+    filter_fields = (
+        'finished_at',
+        'location',
+        'organization',
```

Comment:
what does it mean, to filter on `organization` here? we already filter on `organization__id` and `organization__name`.

Reply:
There is also possible to use query 'organization=id' which is the same with 'organization__id=id'. I don't know whether we need both. Right now I'm not sure whether filtering works as I wanted, so I need to check it in details.

Reply:
With organization, it is possible to filter offers with 'organization=id' query, but I don't know whether we need it. 'organization__id=id' gives the same result. For an organization-details view, I would prefer to utilize this second endpoint in OrganizationViewSet. Filtering here, for offers, can be used with all active and all archived offers views and later if we introduce some search functionality. We should decide what fields can be used for filtering purpose and add here.